src/src/Helpers/HTTPResponseParser.cpp

```diff
@@ -0,0 +1,243 @@
+#include "../Helpers/Networking.h"
```

Comment:
I expected an include of `../Helpers/HTTPResponseParser.h` on the first line.

And then following line to have this:
```c++
#if RESPONSE_PARSER_SUPPORT
```